dynawo/sources/Simulation/DYNSimulation.h

```diff
@@ -592,6 +634,10 @@ class Simulation {
   std::string curvesInputFile_;  ///< curves' resquest input file
   std::string curvesOutputFile_;  ///< curves' output file
 
+  exportFinalStateValuesMode_t exportFinalStateValuesMode_;  ///< final state' export mode
+  std::string finalStateValuesInputFile_;
```

Comment:
missing comments

Reply:
Added comments for new attributes related to final state values.